Github Test.sql

```diff
@@ -1,5 +1,13 @@
+------------------------------------------------------------------------------------

```

Comment:
@tsteiner27, I think it would best to use a different style of comment because it is such a large comment at the beginning of this code. Once you have made this change, I will okay the pull request.

- Matt